util/dvsim/FlowCfg.py

```diff
@@ -339,11 +341,57 @@ def gen_results(self):
         '''Public facing API for _gen_results().
         '''
         results = []
+        result_summary = []
         for item in self.cfgs:
             result = item._gen_results()
             print(result)
             results.append(result)
-        return results
+
+        if self.is_master_cfg: self.gen_results_summary()
+
+    def gen_results_summary(self):
+        '''Public facing API to generate summary results for each IP/cfg file
+        '''
+        return
+
+    def publish_results_summary(self):
+        '''Public facing API for publishing md format results to the opentitan web server.
+        '''
+        results_html_file = 'summarys.html'
```

Comment:
Prefer to call this "summary.html". No need of plural.

Reply:
Done